### DISCUSSION TITLE
eBUS Adapter 5 - No ebus messages logged when making changes via wireless sensoCOMFORT VRC 720f

### DISCUSSION THREAD

**Discussion by VivLloyd** [john30/ebusd#1158] [Q&A]
I've been using the eBUS Adapter 5 in combination with a Raspberry Pi Model B successfully for several months to monitor the performance of a Vaillant Arotherm Plus 5kW Air Source Heat Pump. I would now like to correct some known messages that don't decode correctly using the existing ebusd message files. When I read or write commands such as z1NightTemp (setback temperature) using ebusctl, the ebus messages are logged as expected. When I do the same thing via the sensoCOMFORT control I don't see any ebus messages logged even at DEBUG logging level. What am I doing wrong?

**Comment by Koky05** [comment]
Data from eBus are not stored online. They are loaded in period. So if you change something via sensoCOMFORT you need to wait till next pulling of data into eBUS Adapter.

**Reply by VivLloyd** [reply]
What I was hoping to do was to view the actual raw message on the ebus so that I could use this to correct the corresponding message defined in the message file. For example, if I read ccTimer.Monday via ebusctl, I will get the following error "ERR: invalid position in decode" Since I can issue the corresponding command via my sensoCOMFORT, viewing the raw data would help me correct the decoding. Is there a way to achieve this? Thanks in advance for any help that you can provide.

**Reply by Koky05** [reply]
You could read data by grabbing them
[Wiki HowTos](https://github.com/john30/ebusd/wiki/HowTos)
